<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;input &lt;font color=red&gt;*&lt;/font&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_lineEdit">
       <property name="placeholderText">
        <string>input meida file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;models-dir&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modeldir_lineEdit">
       <property name="placeholderText">
        <string>models directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>json</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="json_lineEdit">
       <property name="placeholderText">
        <string>export FrameContent as JSON</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>proto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proto_lineEdit">
       <property name="placeholderText">
        <string>export FrameContent as Protobuf</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>inspect proto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inspect_proto_lineEdit">
       <property name="placeholderText">
        <string>inspect a protobuf FrameContent by writing to annotated media</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>proto-video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="protovideo_lineEdit">
       <property name="placeholderText">
        <string>output video file annotated with proto frame content</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>outmedia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outmedia_lineEdit">
       <property name="placeholderText">
        <string>annotated output media file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="4">
      <widget class="QCheckBox" name="ik_checkBox">
       <property name="text">
        <string>use ik</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="green_checkBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>green screen</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="head_checkBox">
       <property name="text">
        <string>head pose</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="track_groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>tracking</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>tracker kind</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="tracker_kind_comboBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;tracker used: V1,V2, None. (default: V2)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <item>
           <property name="text">
            <string>V1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>V2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="hand_checkBox">
       <property name="text">
        <string>hand pose</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="d3d_checkBox">
       <property name="text">
        <string>3d pose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>gpu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gpu_spinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;2d net resolution (default: 456x256)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>net-res</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="net_res_lineEdit">
         <property name="placeholderText">
          <string>2d net resolution (default: 456x256)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="serialize_lineEdit">
       <property name="placeholderText">
        <string>serialize a pose estimator</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="serialize_radioButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;serialize a pose estimator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>serialize (save)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="deserialize_radioButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;deserialize an optimized pose estimator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>deserialize (load)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>fbx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fbx_lineEdit">
       <property name="placeholderText">
        <string>write an FBX animation file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>inspect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inspect_lineEdit">
       <property name="placeholderText">
        <string>inspect a serialized model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>2d model regex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="model_regex_lineEdit">
       <property name="placeholderText">
        <string> regex to select 2d model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>min valid joints</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="min_valid_joint_lineEdit">
       <property name="placeholderText">
        <string>number of valid joints for a pose to be  acccepted</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>wrCLI file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
